### PR TITLE
Reorganised roadmap, add high-level goals

### PR DESCRIPTION
**Pull request by bfirsh**
Following on from https://github.com/docker/fig/pull/765, I've reorganised it to explain what our high-level goals for Figpose are.

Maybe we should leave this pull request open for a bit longer than the last one for feedback.

### CODE REVIEW

**Comment by aanand** [comment]
Much better :+1:

**Comment by bfirsh** [comment]
I have renamed all instances of "Compose" to "Fig", because it's still called Fig. When we rename to Compose, we can update the roadmap.

File: ROADMAP.md [hunk 1]
```diff
@@ -1,20 +1,28 @@
 # Roadmap
 
-## Fig 1.1
+Fig will be incorporated as part of the Docker ecosystem and renamed Docker Compose. The command-line tool tool and configuration file will get new names, and its documentation will be moved to [docs.docker.com](https://docs.docker.com).
```
Comment (xiaods) [inline ROADMAP.md]: tool tool

Comment (bfirsh) [inline ROADMAP.md]: thanks!

**Pull request merged**